config/locales/en.yml

```diff
@@ -250,19 +250,27 @@ en:
     already_exists: This assignment already exists!
     title_too_large: The title shouldn't exceed 255 characters
     assignees: Assignee(s)
-    assign_other: Assign/remove an article
+    assign_other:
```

Comment:
As in the other PR, let's keep these existing message keys as they are and continue using them for non-wikidata, and just add a new ones for wikidata terminology.